raphtory-graphql/src/model/graph/mutable_graph.rs

```diff
@@ -173,16 +165,17 @@ impl GqlMutableGraph {
                 )?;
             }
             let constant_props = edge.constant_properties.unwrap_or(vec![]);
+            let edge_view = self
```

Comment:
This needs to go back inside - if the vec of updates is empty, no need to update embeddings